### DISCUSSION TITLE
Bug terms violation screening bot causing accounts suspension right after connecting to CoPilot in VSCode

### DISCUSSION THREAD

**Discussion by ygetelcare** [community/community#137186] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### GitHub Feature Area

Security and Privacy

### Body

Either the violation scanning bot has an over-zealous logic or bug, but merely connecting to CoPilot in VSCode automatically caused the suspension of my brand new account, yge2020, created just to try CoPilot in VSCode the next day and I did not even get to use CoPilot at all. No resolution nor response after more than three weeks other an a ticket number. How ridiculous, this is technically fraudulent business practice, we need to report them to the BBB.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ygetelcare, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
